docs/core/tutorials/creating-app-with-plugin-support.md

```diff
@@ -8,13 +8,16 @@ ms.date: 10/16/2019
 
 # Create a .NET Core application with plugins
 
-This tutorial shows you how to create a custom <xref:System.Runtime.Loader.AssemblyLoadContext> to load plugins. An <xref:System.Runtime.Loader.AssemblyDependencyResolver> is used to resolve the dependencies of the plugin. The tutorial correctly isolates the plugin's dependencies from the hosting application. You'll learn how to:
+This tutorial shows you how to create a custom <xref:System.Runtime.Loader.AssemblyLoadContext> to load plugins. An <xref:System.Runtime.Loader.AssemblyDependencyResolver> is used to resolve the dependencies of the plugin. The tutorial provides a separate namespace for the plugin's dependencies, allowing different assembly dependencies between the plugins and the hosting application. You'll learn how to:
```

Comment:
What does "namespace" in this sentence refer to? I don't think the C# namespace is what is implied here and using that term is likely to cause more confusion.

Reply:
Namespace meaning that type names do not necessarily resolve to the same types. I really don't know a better runtime term for that.

That is, Newtonsoft.JSON.JsonObject in one context does not necessarily refer to the same type as Newtonsoft.JSON.JsonObject in another ALC.

Reply:
> That is, Newtonsoft.JSON.JsonObject in one context does not necessarily refer to the same type as Newtonsoft.JSON.JsonObject in another ALC.

Right. "Assembly context", as proposed in https://github.com/dotnet/docs/pull/43679#discussion_r1851103877, is what I would use.